configdefinitions/src/vespa/configserver.def

```diff
@@ -52,5 +52,6 @@ ztsUrl string default=""
 nodeAdminInContainer bool default=true
 
 # Maintainers
+maintainerIntervalMinutes int default=30
```

Comment:
Seems strange to have 2 timeouts where one is static and one comes from config? Maybe do it like for node-repository maintainers where we can override with environment variable?

Reply:
Yes, this is preliminary while testing this out. It would not be needed if we had job control (as we have for maintainers in controller) or feature flags. I should have noted that in the code, will do in next PR